examples/AccountManagement/GetAccountHierarchy.php

```diff
@@ -0,0 +1,194 @@
+            ArgumentNames::CUSTOMER_ID => GetOpt::REQUIRED_ARGUMENT
+        ]);
+
+        // Generate a refreshable OAuth2 credential for authentication.
+        $oAuth2Credential = (new OAuth2TokenBuilder())->fromFile()->build();
+
+        // Construct a Google Ads client configured from a properties file and the
+        // OAuth2 credentials above.
+        $googleAdsClient = (new GoogleAdsClientBuilder())->fromFile()
+            ->withOAuth2Credential($oAuth2Credential)
+            ->build();
+
+        try {
+            self::runExample(
+                $googleAdsClient,
+                $options[ArgumentNames::CUSTOMER_ID] ?: self::CUSTOMER_ID
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     */
+    public static function runExample(GoogleAdsClient $googleAdsClient, int $customerId)
+    {
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        // Creates a query that retrieves all child accounts of the specified customer ID.
+        $query = 'SELECT customer_client.client_customer, customer_client.level,'
+            . ' customer_client.manager, customer_client.descriptive_name,'
+            . ' customer_client.currency_code, customer_client.time_zone,'
+            . ' customer_client.id'
+            . ' FROM customer_client';
+
+        $unprocessedManagerAccounts = [$customerId];
+        $customerIdsToChildAccounts = [];
+        $rootCustomerClient = null;
+        while (!empty($unprocessedManagerAccounts)) {
+            $managerAccountId = array_shift($unprocessedManagerAccounts);
+            // Issues a search request by specifying page size.
+            $response = $googleAdsServiceClient->search(
+                $managerAccountId,
+                $query,
+                ['pageSize' => self::PAGE_SIZE]
+            );
+
+            // Iterates over all rows in all pages and prints the requested field values for
```

Comment:
You may want to change this comment, as this loop doesn't print anything.

Reply:
Done.